src/app/public/modules/modal/modal-host.component.ts

```diff
@@ -70,11 +70,6 @@ export class SkyModalHostComponent {
     private changeDetector: ChangeDetectorRef
   ) { }
 
-  @HostListener('click', ['$event'])
```

Comment:
You'll need to remove `HostListener` from your import statements above for this to build.